screens/ChatScreen.js

```diff
@@ -47,68 +51,55 @@ class ChatScreen extends Component {
   state = {
     hmToken: {},
     messages: [],
+    loading: true,
   };
 
   async componentDidMount() {
-    console.log('Chat screen');
-
     const token = await AsyncStorage.getItem('hm_token');
     this.setState({
       hmToken: token
     });
 
     this.state.id = `${JSON.parse(token)._id}`;
 
-    this.getTwilioToken().then((twilioToken) => {
-      this.initChatClient(twilioToken).catch((error) => {
-        console.log(error);
+    await this.getTwilioToken().then((twilioToken) => {
```

Comment:
These aren't really how to use await. Awaits are used to get rid of promises to make code more linear. In this case it would be 
```
try {
const token  = await this.getTwilioToken();
this.initChatClient(token);
}
catch (error) {
// log
}

this.setState({loading: false});
```

Reply:
(Usually just take whatevers in the .then and put it in the try block with the await, and put what evers in the .catch in the catch block, and get rid of the .then and .catch)

Reply:
That makes sense. I think I have the appropriate structure now.